recipe/meta.yaml

```diff
@@ -33,7 +33,6 @@ requirements:
   host:
     - capnproto
     - cython
-    - libxcrypt  # [py==38 and linux]
```

Comment:
When adding the Python 3.13 migrator, we dropped Python 3.8 ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6344 ). It also went [EOL]( https://devguide.python.org/versions/ ) earlier this month. So we can drop this workaround